cypress/integration/pagination.spec.ts

```diff
@@ -96,6 +146,8 @@ describe('Alert', () => {
   it('should emit page on page select', () => {
     prepareComponent(3, 5);
 
+    cy.wait(200);
```

Comment:
Is dit echt nodig?

Het zou kunnen dat `prepareComponent` iets asynchroons doet waardoor de test inderdaad 'te vroeg is'. Ik denk dat we dit beter anders kunnen oplossen door bijvoorbeeld prepareComponent te awaiten of op te nemen in de test-chain.

Reply:
Omdat we nu gebruik maken van de responsive element en de size daarvan pakken in de `componentDidLoad` is er een `wait` nodig voordat dit allemaal goed is gerenderd. Dit hindert alleen wanneer het aantal paginas boven het aantal posities valt. 

Lijkt mij het handigst om deze `wait` dan in de `prepareComponent` te zetten, zodat hier niet meer aan gedacht hoeft te worden bij het schrijven van verdere tests.

Reply:
Aha. ik snap het! Goed idee.